### DISCUSSION TITLE
Problems with the Fn function key on the asus rog zephyrus g14 GA401QM laptop

### DISCUSSION THREAD

**Discussion by JarkiyZad** [Atlas-OS/Atlas#1171] [Help & Support]
The computer stopped responding to the Fn+F1-F12 combinations, and the website https://key-test.ru/ also does not respond to this key. Half a year ago I already had the same problem, i.e. everything was working fine and at one point the laptop just stopped responding to this key. 
Neither the installation of Asus keyboards hotkeys, nor the ASUS ATK ACPI Driver, nor all the drivers helped, last time everything was fixed by reinstalling OC

**Comment by trolljoe** [comment]
your laptop might have a f lock key

**Reply by JarkiyZad** [reply]
So it was blocked by the combination win+fn or win+alt (I don't remember which one exactly), but now it doesn't work

**Comment by he3als** [comment]
Hi, can you try enabling Windows Update drivers?
https://docs.atlasos.net/getting-started/post-installation/drivers/getting-started/#want-driver-installation-updates-through-windows-update-instead
